superheroes.yaml

```diff
@@ -137,6 +139,7 @@ scripts:
         ${{ CONTAINER_RUNTIME_PARAMS }}
         --memory ${{HEROES_REST_MEMORY}}
         --cpus ${{HEROES_REST_CPU}}
+        --user "$(id -u):$(id -g)"
```

Comment:
why do we need to map user:group in the container?

Reply:
This is actually required when using `docker` in order to make use of `pmap`. Otherwise the /proc/PID/smaps file would be owned by root and pmap will not have enough permission to access it and I would avoid to use sudo cmds. Do you see any issue in forcing the user:group?

Reply:
I made it configurable because for instance for `jvm` builds they are specifying the USER in the Dockerfile so we cannot use our own user to startup the container